optimum/intel/utils/import_utils.py

```diff
@@ -113,6 +113,17 @@
         _diffusers_available = False
 
 
+_open_clip_available = importlib.util.find_spec("open_clip") is not None
+_open_clip_version = "N/A"
+if _open_clip_available:
+    try:
+        import open_clip
+
+        _open_clip_version = open_clip.__version__

```

Comment:
why not 

```suggestion
        _open_clip_version = importlib_metadata.version("open_clip_torch")
```

Reply:
open_clip.__version__  is not work for open_clip

Reply:
thanks for the update, `_open_clip_version = importlib_metadata.version("open_clip_torch")` works for me, could you extend on when this won't work as expected ?